root/static/scripts/common/components/EditorLink.js

```diff
@@ -18,11 +18,16 @@ const EditorLink = ({editor, content, avatarSize, subPath}) => {
     avatarSize = 12;
   }
 
-  const gravatar = editor.gravatar + '&s=' + (avatarSize * 2);
+  let gravatar;
+  if (editor.gravatar) {
+    gravatar = editor.gravatar + '&s=' + (avatarSize * 2);
```

Comment:
The editor classs knockout uses doesn't store gravatars, so I'm making that prop optional here.